### PR TITLE
Home Task: CSS_02

### PR DESCRIPTION
**Pull request by glebpigulevsky**

### CODE REVIEW

File: gpigulevsky/CSS_02/CSS_03_03.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+		
+		img.icon{
+					width: 128px;
+			height: 128px;
+		}
+		img.align-right{
+			float: right;
+		}
+		.wmail {
+			padding-bottom: 140px;
```
Comment (pluhin) [inline gpigulevsky/CSS_02/CSS_03_03.html]: What is that? You need to "include" picture using another approach

**Review by pluhin — CHANGES_REQUESTED** [review]
See comments 
+
could you please do rebase

File: gpigulevsky/CSS_02/CSS_03_03.html (hunk 1 above)
Comment (glebpigulevsky) [reply gpigulevsky/CSS_02/CSS_03_03.html]: fixed

File: gpigulevsky/CSS_03/style_menu_auto.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            background-color: white;
+            color: gray;
+          }
+          @media all and (min-width: 920px) {
+                      ul {
+                          width: auto;
+                          border-bottom: none;
+                      }
+                      li {                                    /* расположение списка по горизонтали */
+                          display: inline-block;
```
Comment (pluhin) [inline gpigulevsky/CSS_03/style_menu_auto.css:41]: why have you used two? Should be enough just one, or `float` or `display`

**Review by pluhin — CHANGES_REQUESTED** [review]

File: gpigulevsky/CSS_03/style_menu_auto.css (hunk 2 above)
Comment (glebpigulevsky) [reply gpigulevsky/CSS_03/style_menu_auto.css:41]: without float, elements of menu have intervals

Comment (glebpigulevsky) [reply gpigulevsky/CSS_03/style_menu_auto.css:41]: > why have you used two? Should be enough just one, or `float` or `display`

https://files.slack.com/files-pri/TC8AF2ZKL-FQBTB4Q68/____________.png

**Review by pluhin — APPROVED** [review]

**Pull request merged**